src/Text.js

```diff
@@ -24,6 +31,11 @@ class Text
         s += `1\n${this.value}\n`;
         s += `10\n${this.x1}\n20\n${this.y1}\n30\n0\n`;
         s += `40\n${this.height}\n50\n${this.rotation}\n`;
+        if (H_ALIGN_CODES.includes(this.hAlign, 1) | V_ALIGN_CODES.includes(this.vAlign, 1)){
```

Comment:
Use logical operator || here. Actually we should use && to be sure that both params are possible.
...Well this actually always add in the extra group codes since horizontalAlignment  and verticalAlignment have default values.

We could use null as default parameters and here write:
`if (H_ALIGN_CODES.includes(this.hAlign, 1) && V_ALIGN_CODES.includes(this.vAlign, 1))`
Would that work?

Reply:
I can change the logical operator.

I don't think the other changes are necessary because of the following sequence:
1. Nothing passed in for alignment
2. Both alignment parameters are default (`left` and `baseline`)
3. The `includes` is skipping the first element (the default one) and evaluates to `false`
4. The conditional is false and no extra parameters are written.

You can verify this by seeing that the output of the first text in `./examples/text.dxf' does not have the extra lines written.

In the case that you only pass horizontal alignment, the vertical alignment will just be written in as it's default value, but at this point you've already changed the output, so it shouldn't be an issue?

However, it could be changed to `null` as the default as you say and then the `includes` can just check the entire array without skipping the first element.

I'll hold off on the operator change until I have additional feedback on the second point.

Reply:
Yes, you are right I overthought a little bit :).
Just change to logic operator and that would be fine.

Reply:
Ok. Will do!

I'll also just note, for your reference, incorrect operator is the one type of error that can be caught automatically by TypeScript.

`The '|' operator is not allowed for boolean types. Consider using '||' instead.ts(2447)`